IBPSA/Fluid/Chillers/BaseClasses/Carnot.mo

```diff
@@ -50,6 +50,14 @@ partial model Carnot
     "Pressure difference over evaporator"
     annotation (Dialog(group="Nominal condition"));
 
+  parameter Modelica.SIunits.TemperatureDifference TAppCon_nominal(min=0) = if cp1_default < 1500 then 5 else 2
```

Comment:
It would make sense to let the name reflect that the parameter is a temperature _difference_?
E.g. use `dT` or `TDif`